proto/sequencerblockapis/astria/sequencerblock/v1alpha1/optimistic_block.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+
+package astria.sequencerblock.v1alpha1;
+
+import "astria/primitive/v1/types.proto";
+import "astria/sequencerblock/v1alpha1/block.proto";
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
+
+// The "BaseBlock" is the information needed to simulate bundles on top of
+// a Sequencer block which may not have been committed yet.
+message BaseBlock {
```

Comment:
this should be in the bundle.proto file

Reply:
Agreed on should be in the bundle.proto, open question of should we use the `ExecuteBlockRequest` shape from execution API?

Reply:
> Agreed on should be in the bundle.proto, open question of should we use the ExecuteBlockRequest shape from execution API?

The bundle api's `BaseBlock` is the same as the execution api's `ExecuteBlockRequest` except for the fact that it contains the sequencer block hash rather than the previous rollup hash.

The reason for having the sequencer block hash is that the BaseBlock is derived from a FilteredSequencerBlock  (thus has the corresponding sequencer hash), and we want to link the execution result to a sequencer block commitment that will received later (so we provide the sequencer hash to geth's bundle service).

The reasons we don't have the previous rollup block hash in the auctioneer are:
1. Getting it will require calling GetCommitmentState  which will incur additional latency
2. We dont actually care about the previous rollup block hash in the auctioneer, just about the block hash that results from the optimistic block execution (i.e. the rollup block hash sent in response)

Reply:
yeah agreed with itamar. We don't want to call `GetCommitmentState` in auctioneer as it ll add to latency. I am a little unsure about requiring `sequencer_block_hash` in the geth node. I think auctioneer could maintain the session by itself

Reply:
The auctioneer will be managing this state regardless in order to drive the auction.

Without adding this into the requests and responses we rely on both in-order arrival of messages over the stream and the no delays/drops that could fuck things up. Feels easier to just have this as part of the request/response, but there's no need to keep track of it inside of geth since the request should be in scope when u make the response?